x-pack/plugins/cases/public/components/user_actions/property_actions/property_actions.tsx

```diff
@@ -13,9 +13,14 @@ import { PropertyActions } from '../../property_actions';
 interface Props {
   isLoading: boolean;
   propertyActions: PropertyActionButtonProps[];
+  customDataTestSubj?: string;
 }
 
-const UserActionPropertyActionsComponent: React.FC<Props> = ({ isLoading, propertyActions }) => {
+const UserActionPropertyActionsComponent: React.FC<Props> = ({
+  isLoading,
+  propertyActions,
+  customDataTestSubj = 'user-action',
```

Comment:
Let's default all the user action property actions to `user-action`. In the future we may want to make these more specific. One particular scenario is the description so that some of the functional tests can be more specific.